src/trans/gpu/external/setup_trans0.F90

```diff
@@ -125,7 +126,6 @@ SUBROUTINE SETUP_TRANS0(KOUT,KERR,KPRINTLEV,KMAX_RESOL,KPROMATR,&
 MYGPU = MOD(MYPROC-1,NUMDEVS)
 CALL ACC_SET_DEVICE_NUM(MYGPU, IDEVTYPE)
 MYGPU = ACC_GET_DEVICE_NUM(IDEVTYPE)
-WRITE(*,*) 'MYPROC:',MYPROC, 'GPU:', MYGPU, 'of ', NUMDEVS
```

Comment:
Because we always pin one GPU for each rank, this only ever prints
```
MYPROC: <whatever> GPU: 0 of 1
```
Which is not that useful... Unless there's a risk of accidentally pinning multiple GPUs for each rank. That sounds like  it would be a good check actually. We could abort if that's the case.

Reply:
If you don't do the per-rank pinning, ectrans will set the GPU device corresponding to the current rank here. So, if you have e.g. `CUDA_VISIBLE_DEVICES=0,1,2,3`, then you should get output like `0 of 4`, `1 of 4` etc. For over-subscription you get round-robin.
But I agree that this might be enough as a debug output

Reply:
Since it's only printed once, it's harmless. Let's add it back!

Reply:
We could have a ECTRANS_DEBUG environment variable to control debugging output?
This is how many packages in the stack configure their debug output.